packages/page-bounties/src/Bounties.tsx

```diff
@@ -51,8 +56,56 @@ function Bounties (): React.ReactElement {
             />
           ))}
       </Table>
-    </>
+    </div>
   );
 }
 
-export default React.memo(Bounties);
+export default React.memo(styled(Bounties)`
+  .bounties-table-wrapper table {
+    background: none;
+
+    &::before {
+      background: none;
+    }
+
+    tr {
+
+      td,
+      &:not(.filter) th {
+        &:first-child {
+          padding-left: 1.14rem;
+        }
+
+        &:last-child {
+          padding-right: 1.14rem;
+        }
+      }
+    }
+  }
+  
+  .ui--IdentityIcon {
+    margin-right: 0.42rem;
+  }
+
+  .via-identity .name {
+    font-size: 1rem;
+    line-height: 1.7rem;
+    text-transform: initial;
+    filter: initial;
+    opacity: 1;
+  }
+
+  .ui--Table th:first-child h1 {
+    padding: 0;
+    font-weight: bold;
+    font-size: 0.7rem;
+    line-height: 0.85rem;
+    text-transform: uppercase;
+    color: #4D4D4D;
```

Comment:
Not sure this is intentional? It is different from all other headers and will mean we need to circle back when making any updates?